webapp/packages/core-connections/src/ConnectionInfoResource.ts

```diff
@@ -18,11 +18,12 @@ import {
   CachedMapResource,
   ConnectionConfig,
   UserConnectionAuthPropertiesFragment,
-  resourceKeyList
+  resourceKeyList,
 } from '@cloudbeaver/core-sdk';
 
 import { ConnectionsResource } from './Administration/ConnectionsResource';
 import { CONNECTION_NAVIGATOR_VIEW_SETTINGS } from './ConnectionNavigatorViewSettings';
+import type { DBAuthPartialConfig } from './DatabaseAuthDialog/DBAuthDialogController';
```

Comment:
seems `init` and `initConnection` function should accept `IDBAuthConfig` as object